pontoon/base/forms.py

```diff
@@ -172,7 +172,8 @@ class Meta:
 
     def clean_email(self):
         email = self.cleaned_data.get('email')
-        if email and User.objects.filter(email=email).exclude(username=self.instance.username).exists():
+        if email and User.objects.filter(email=email)\
+                .exclude(username=self.instance.username).exists():
```

Comment:
Please write it like this instead: 

```python
if (
    email and
    (
        User.objects.filter(email=email)
        .exclude(username=self.instance.username)
        .exists()
    )
):
```

All of those comments are for the sake of consistency of coding style across our code base.